uwblueprint/website-bp
# Clean up application flow UX

### Implementation description

Again, strongly recommend reviewing by commit and reading the commit description where applicable.

Primary changes:
- Turn the first dev question into a multi-select question
<img width="864" alt="image" src="https://user-images.githubusercontent.com/9922514/219854786-89fff154-df05-46b0-b779-e1eff193a12d.png">


Additional changes resulting from discussions on Slack:
- Allow non-required selects to have their values cleared
<img width="496" alt="image" src="https://user-images.githubusercontent.com/9922514/219854800-2de0658d-03d8-4c90-8dd2-52e20edfd305.png">

- Update "Our Process" copy based on Slack discussions
<img width="578" alt="image" src="https://user-images.githubusercontent.com/9922514/219854813-c9c45091-d4b3-4fd3-beed-5ebb3a4c6a24.png">

- Make links throughout the site more clear (both visually and markup-wise, see 5deeba72449867531e4ff2f0202aa760393be81e commit message for details)
<img width="459" alt="image" src="https://user-images.githubusercontent.com/9922514/219854847-fe3ad725-9bdc-4024-864e-2a996b625bae.png">


Miscellaneous changes:
- Remove unused resource file which should have been deleted a while ago



### Steps to test

Primary changes:
1. Create a dev application.
2. Verify it's submittable and looks good.
3. Verify it's viewable on the admin side.

Additional changes:
1. Test selects by deselecting values in the application flow.
2. View the copy on the "Join" page.
3. Hover various links on the site and verify they look good.
4. Tab through various pages and verify the tab flow is as expected.

Miscellaneous changes:
1. (nothing to test, if it builds successfully we should be good)




### What should reviewers focus on?

- Do the visual changes look good given the time constraints on this PR being merged?
- Is the code clean?

### Checklist
